sds_data_manager/utils/trigger_bulk_reprocessing.py

```diff
@@ -0,0 +1,165 @@
+#! /usr/bin/env python3
+"""Triggers the api endpoint to reprocess the data."""
+
+# ruff : noqa: S310
+import argparse
+import contextlib
+import logging
+import urllib.request
+from typing import Optional
+from urllib.error import HTTPError, URLError
+from urllib.parse import urlencode
+
+# Logger setup
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+
+@contextlib.contextmanager
+def _get_url_response(url: str):
```

Comment:
If we are already importing imap-data-access can you use the code there which has been refactored to use requests now?